### PR TITLE
Bacpop-115 About-page + Duplicate name creation/deletion

### PR DESCRIPTION
**Pull request by absternator**
About Page... Words copied straight from ticket..

Splipped in a  quick line to not let users create/edit duplicate project names

### CODE REVIEW

File: app/client-v2/src/views/AboutView.vue [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 <template>
-  <div class="p-5 mb-3 border-round-sm surface-card">About Page</div>
+  <div class="p-5 mb-3 border-round surface-card shadow-2" style="max-width: 900px">
+    <h1 class="text-2xl font-semibold">About</h1>
+    <p class="text-lg">
+      BeeBOP is being developed by the RESIDE group within MRC GIDA as an online interface to the PopPUNK method
+      developed by John Lees (EMBL-EBI) and Nick Croucher (Imperial College London).<br />
+      This work is part of the Global Pneumococcal Sequencing project, led by Stephen Bentley (Wellcome Sanger
+      Institute) and Lesley McGee (CDC), managed by Stephanie Lo (Wellcome Sanger Institute), and funded by the Bill &
+      Melinda Gates Foundation and Welcome.
```
Comment (EmmaLRussell) [inline app/client-v2/src/views/AboutView.vue:11]:
```suggestion
      Melinda Gates Foundation and Wellcome.
```

**Review by EmmaLRussell — COMMENTED** [review]
Could make the org references into links I guess. Could you check with Nick if they want logos for the various orgs on the app, and if so can they go on the About page?

**Comment by absternator** [comment]
> Could make the org references into links I guess. Could you check with Nick if they want logos for the various orgs on the app, and if so can they go on the About page?

Good idea checked with nick and he said good idea too so done

File: app/client-v2/src/views/AboutView.vue [hunk 2]
```diff
@@ -1,14 +1,55 @@
...
+          rel="noopener noreferrer"
+          class="no-underline text-primary hover:underline"
+          >CDC</a
+        >), managed by Stephanie Lo (Wellcome Sanger Institute), and funded by the
+        <a
+          href="https://www.gatesfoundation.org/"
+          target="_blank"
+          rel="noopener noreferrer"
+          class="no-underline text-primary hover:underline"
+          >Bill & Melinda Gates Foundation and Wellcome.</a
```
Comment (EmmaLRussell) [inline app/client-v2/src/views/AboutView.vue:43]: Wellcome should be a separate link presumably!

**Review by EmmaLRussell — COMMENTED** [review]
I don't think you've used the added Image component? Looks like ordinary img tags below.

Should disallow whitespace only names too.

File: app/client-v2/src/views/AboutView.vue (hunk 2 above)
Comment (absternator) [reply app/client-v2/src/views/AboutView.vue:43]: haha yeah i thought it was one word my bad

**Review by EmmaLRussell — APPROVED** [review]
Still looks like you're not using that Image component?
Good to go otherwise! :+1:

**Comment by absternator** [comment]
> Still looks like you're not using that Image component?

ohh sorry missed this.. but the image component was having some issues with the paths so used the regular img tag

**Pull request merged**